app/src/userpages/components/StreamPage/Show/index.jsx

```diff
@@ -264,11 +241,67 @@ export class StreamShowView extends Component<Props, State> {
     }
 }
 
+function StreamLoader(props: Props) {
+    const isMounted = useIsMounted()
+    const propsRef = useRef(props)
+    propsRef.current = props
```

Comment:
`props` is set as the initial value before, no? Does this update only once?

Reply:
Flow wants an initial value passed to `useRef` otherwise it thinks `propsRef` can be `undefined`.